src/test/java/guitests/guihandles/GuiHandle.java

```diff
@@ -28,15 +33,39 @@ public GuiHandle(GuiRobot guiRobot, Stage primaryStage) {
     public <T> T as(Class<? extends GuiHandle> clazz) {
         try {
             Constructor<?> ctor = clazz.getConstructor(GuiRobot.class, Stage.class);
-            Object object = ctor.newInstance(new Object[] { guiRobot, primaryStage });
+            Object object = ctor.newInstance(new Object[] { guiRobot, primaryStage});
             return (T)object;
         } catch (Exception e) {
             throw new RuntimeException("Cannot create gui handle of type " + clazz.getName(), e);
         }
     }
 
+    public void focusOnWindow(String stageTitle) {
+        LoggerManager.getLogger(this.getClass()).info("Focusing " + stageTitle);
```

Comment:
Logger could extracted into a constant. It should still keep the reference to the calling object's type. (See `ChangeObjectInModelCommand`)